File: ShipperManagerWeb/ShipperManagerWeb/ShipperManager/Models/Shipper.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace ShipperManager.Models
{
    public class Shipper
    {
        public string Id { get; set; }
        [Display(Name = "Tên")]
        public string Ten { get; set; }
        [Display(Name = "Quê Quán")]
        public string QueQuan { get; set; }
        [Display(Name = "Tài Khoản")]
        public string TaiKhoan { get; set; }
        [Display(Name = "Mật Khẩu")]
        public string MatKhau { get; set; }
        [Display(Name = "Ngày Sinh")]
        public string NgaySinh { get; set; }
        [Display(Name = "Trạng Thái")]
        public bool TrangThai { get; set; }
        [Display(Name = "Ảnh")]
        public string ImagePath { get; set; }
    }
}

File: ShipperManagerAndroid/ShipperManagerAndroid/app/src/main/java/com/example/shippermanager/Model/Shipper.java
package com.example.shippermanager.Model;

public class Shipper {

    public String Id;
    public String Ten;
    public String NgaySinh;
    public String QueQuan;
    public String TaiKhoan;
    public String MatKhau;
    public boolean TrangThai;
    public double KinhDo;
    public double ViDo;
    public String ImagePath;
    public Kho Kho;

    public Shipper() {

    }

    public Shipper(String id, String ten, String ngaySinh, String queQuan, String taiKhoan, String matKhau, boolean trangThai, double kinhDo, double viDo,String imagePath) {
        Id = id;
        Ten = ten;
        NgaySinh = ngaySinh;
        QueQuan = queQuan;
        TaiKhoan = taiKhoan;
        MatKhau = matKhau;
        TrangThai = trangThai;
        KinhDo = kinhDo;
        ViDo = viDo;
        ImagePath = imagePath;
    }
    public Shipper(String id, String ten, String ngaySinh, String queQuan, String taiKhoan, String matKhau, boolean trangThai, double kinhDo, double viDo,String imagePath,Kho kho) {
        Id = id;
        Ten = ten;
        NgaySinh = ngaySinh;
        QueQuan = queQuan;
        TaiKhoan = taiKhoan;
        MatKhau = matKhau;
        TrangThai = trangThai;
        KinhDo = kinhDo;
        ViDo = viDo;
        ImagePath = imagePath;
        Kho =kho;
    }
}
